layers/+lang/lua/packages.el

```diff
@@ -30,17 +30,10 @@
     :interpreter ("lua" . lua-mode)
     :init
     (progn
-      (setq lua-indent-level 2
-            lua-indent-string-contents t)
-      (spacemacs/set-leader-keys-for-major-mode 'lua-mode
-        "d" 'lua-search-documentation
-        "sb" 'lua-send-buffer
-        "sf" 'lua-send-defun
-        "sl" 'lua-send-current-line
-        "sr" 'lua-send-region))))
```

Comment:
You should add a binding to "'" to start a repl process as well as add a repl registration like in 
```elisp
      (spacemacs/register-repl 'python
                               'spacemacs/python-start-or-switch-repl "python")
```
For reference you should check the python layer.